modules/odm_georef/src/Georef.hpp

```diff
@@ -46,10 +46,11 @@ struct GeorefGCP
 
     size_t cameraIndex_;    /**< The index to the corresponding camera for the image. **/
 
-    int pixelX_;            /**< The pixels x-position for the GCP in the corresponding image **/
-    int pixelY_;            /**< The pixels y-position for the GCP in the corresponding image **/
+    double pixelX_;            /**< The pixels x-position for the GCP in the corresponding image **/
```

Comment:
Good catch, since GCPs support floats we have been effectively truncating the precision here.